en/docs/develop-components/develop-integrations/develop-a-scheduled-task.md

```diff
@@ -1,113 +1,101 @@
-# Develop a Scheduled Task
+# Develop a scheduled task
 
-Choreo is a versatile integration platform that allows you to create various types of integrations depending on your requirement. 
+Choreo is a versatile integration platform that allows you to create various types of integrations depending on your requirements. If you need to automatically run a specific integration at regular intervals, you can use Choreo to develop a scheduled task. This type of integration can automate the synchronization of data between different systems at specified intervals, reducing errors and improving productivity by eliminating the need for manual intervention.
 
-If you have a requirement to automatically run a specific integration at regular intervals, you can use Choreo to develop a scheduled task. This type of integration can automate the synchronization of data between different systems at specified intervals, reducing errors and improving productivity by eliminating the need for manual intervention.
-
-This guide walks you through the steps to develop, deploy, test, and observe a scheduled task using Choreo.  
+This guide walks you through the steps to develop, deploy, test, and observe a scheduled task using Choreo.
 
 In this guide, you will:
 
-  - Develop a scheduled task to fetch the weather forecast for a specified location for the next 24 hours from the [OpenWeatherMap](https://openweathermap.org) API.
-  - Process the weather data to a specific format.
-  - Send the formatted data to a specified email address at a specific time every day.
+- Develop a scheduled task to fetch the weather forecast for a specified location for the next 24 hours from the [OpenWeatherMap](https://openweathermap.org) API.
+- Process the weather data into a specific format.
+- Send the formatted data to a specified email address at a specific time every day.
 
 ## Prerequisites
 
-Before you try out the steps in this guide, complete the following:
+Before you begin:
 
- - If you are signing in to the Choreo Console for the first time, create an organization as follows:
-    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in using your preferred method.
-    2. Enter a unique organization name. For example, `Stark Industries`.
+1. **Create a Choreo Organization**:
```

Comment:
We generally don't use bold formatting and title caps in step-by-step instructions. See https://developers.google.com/style/text-formatting and https://github.com/wso2/docs-choreo-dev/wiki/Markdown-File-Naming-Conventions-and-Content-Formatting#text-formatting-guidelines.